web/rootfs/defaults/meet.conf

```diff
@@ -22,6 +22,10 @@ add_header X-XSS-Protection "1; mode=block";
 add_header X-Jitsi-Shard {{ .Env.DEPLOYMENTINFO_SHARD }};
 {{ end }}
 
+{{ if .Env.ENABLE_FLOC | default "0" | toBool }}
+add_header Permissions-Policy "interest-cohort=()";
```

Comment:
AFAIS in https://plausible.io/blog/google-floc the header must be set to opt-out, so the if needs to be reversed, and set the header when the enabled flag has NOT been set.

Reply:
Thank you for spotting this, I missed that while inverting the logic. I hope I got the frep/golang syntax correct in efc3f30